scripts/start/readCheckScenarioConfig.R

```diff
@@ -131,3 +116,25 @@ readCheckScenarioConfig <- function(filename, remindPath = ".", testmode = FALSE
   if (errorsfound > 0) if (testmode) warning(errorsfound, " errors found.") else stop(errorsfound, " errors found, see explanation in warnings.")
   return(scenConf)
 }
+
+copyConfigFrom <- function(scenConf) {
```

Comment:
Our [coding etiquette](https://github.com/remindmodel/remind/blob/develop/tutorials/07a_Advanced_ChangeStartingScripts.md#coding-etiquette-structuring) says:
> one file per function (you can have multiple functions in one file, but only the topmost should be used from other files), file name is the same as function name

So, I think you should move copyConfigFrom into `copyConfigFrom.R`.

Reply:
Why is "you can have multiple functions in one file, but only the topmost should be used from other files" not satisfied?

Reply:
It is, I'm just confused. Thanks for double-checking.